dist/materia/_bootswatch.scss

```diff
@@ -354,10 +354,11 @@ select.form-control {
   &:checked {
     ~ .custom-control-label::before {
       background: $white;
+      transition: border-color .24s;
```

Comment:
I'm not sure about this change TBH. Can you verify if this is needed? Maybe the next change needs some tweaking but I generally tried to keep things simple.

Reply:
I checked (no pun intended) and it's needed to set the transition duration to the same `::after`. The default being .15s.

Reply:
If you triple checked and everything looks the same, feel free to merge :)